.github/workflows/deploy.yml

```
@@ -7,4 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Dump event
-        run: jq . $GITHUB_EVENT_PATH
\ No newline at end of file
+        run: jq . $GITHUB_EVENT_PATH
```

Rather than just dumping the GitHub event, I'd like to change this to something that shows the originating repository. It would be something like:

```suggestion
        run: echo {{github.event.client_payload.repo}}
        if: github.event.action == 'build_application'
```

and change `Dump event` to `Display triggering repository` or something.

This could be overkill. So maybe let's just think about this as an enhancement.

---

See https://github.com/multinet-app/multinet-deploy/issues/24